src/app/components/iterations-panel/iterations-panel.component.less

```diff
@@ -1,92 +1,24 @@
 @import '../../../assets/stylesheets/_base';
-
+.add-iteration {
+  margin-top: -58px;
+  float: right;
+}
 .iterations-container {
   background-color: @color-pf-black-900;
   .iteration-wrapper {
     color: @color-pf-white;
-    li { cursor: pointer; }
-    .badge { background-color: @color-pf-black; }
-    i { color: @color-pf-white; }
-    .iteration-header { padding: 10px 20px;
-      > h4 { margin: 0; }
-      .fa-icon { width: 15px; }
-      .add-btn { color: @color-pf-blue; }
-    }
-    .iteration-subheader {
+  }
+  .iterations {
+    .list-group-item,
+    .tree-list-item {
+      background-color: @color-pf-black-900;
```

Comment:
from the inspector, it doesn't look like this is being called. You can try
```
side-panel {
  .tree-list {
    .list-group {
      .list-view-pf {
        .list-group-item {
          background-color: @color-pf-black-900;
        }
      }
    }
  }
}
```
and that will be very specific to only target the `.list-group-item` inside of the side-panel